barretenberg/cpp/src/barretenberg/ultra_honk/relation_correctness.test.cpp

```diff
@@ -267,13 +267,13 @@ TEST_F(RelationCorrectnessTests, UltraRelationCorrectness)
     auto circuit_size = proving_key->circuit_size;
 
     // Generate eta, beta and gamma
-    FF eta = FF::random_element();
-    FF beta = FF::random_element();
-    FF gamma = FF::random_element();
+    instance->relation_parameters.eta = FF::random_element();
+    instance->relation_parameters.beta = FF::random_element();
+    instance->relation_parameters.gamma = FF::random_element();
 
-    instance->initialize_prover_polynomials();
-    instance->compute_sorted_accumulator_polynomials(eta);
-    instance->compute_grand_product_polynomials(beta, gamma);
+    instance->proving_key->compute_sorted_accumulator_polynomials(instance->relation_parameters.eta);
+    instance->proving_key->compute_grand_product_polynomials(instance->relation_parameters);
+    instance->prover_polynomials = Flavor::ProverPolynomials(instance->proving_key);
```

Comment:
we need to update prover_polynomials after this since we need the shifts for relation checking.